etc/Grapl Provision.ipynb

```diff
@@ -149,11 +158,23 @@
     "            schema.add_edge(predicate_meta[\"predicate\"], predicate, \"\")\n",
     "\n",
     "\n",
+    "def store_schema(table, schema: \"BaseSchema\") -> None:\n",
+    "    for f_edge, (_, r_edge) in schema.get_edges().items():\n",
+    "        if not (f_edge and r_edge):\n",
+    "            continue\n",
```

Comment:
I wonder if we should add some logging here (just a print statement, perhaps) that says "hey we found an edge that is missing a reverse edge"

I guess we can just add that as-needed if that ever came up again, but it could be a subtle bug in the future that we don't notice until really deep in the debugging process.